rocks-db/src/clients/asset_client.rs

```diff
@@ -292,10 +301,11 @@ impl Storage {
                 .get_editions(
                     assets_data
                         .values()
-                        .filter_map(|a: &crate::asset::AssetCompleteDetails| {
+                        .filter_map(|a: &AssetCompleteDetails| {
                             a.static_details.as_ref().map(|s| s.edition_address)
                         })
                         .flatten()
+                        .dedup()
```

Comment:
are we sure that this data is sorted when we dedup it? cause it will only remove consecutive equal elements

Reply:
Yes, it is not necessary here. Above we use hashmap for it.